src/delta5node/delta5node.ino

```diff
@@ -269,14 +279,10 @@ void loop() {
 		}
 
 		if (state.crossing) {
-			int triggerThreshold = settings.triggerThreshold;
 
-			// If in calibration mode, keep raising the trigger value
-			if (state.calibrationMode) {
-				state.rssiTrigger = max(state.rssiTrigger, state.rssi - settings.calibrationOffset);
```

Comment:
The intent of auto calibration is to calculate a new trigger value for each new run.  Each run may have a different pilot with different vtx output parameters.